pkgs/tools/security/krunner-pass/default.nix

```diff
@@ -0,0 +1,36 @@
+{ mkDerivation, stdenv,
+  fetchFromGitHub,
+  cmake, extra-cmake-modules, gnumake,
+
+  pass, pass-otp ? null, krunner,
+}:
+let
+  pname = "krunner-pass";
+  version = "v1.3.0";
```

Comment:
The version in the package name should only contain digits (nixpkgs coding conventions).